packages/maskbook/src/plugins/Trader/apis/bancor/calculateMinimumReturn.ts

```diff
@@ -0,0 +1,19 @@
+import type { FungibleTokenDetailed } from '@masknet/web3-shared'
+import BigNumber from 'bignumber.js'
+import { pow10 } from '@masknet/web3-shared'
+import { BIPS_BASE } from '../../constants'
+
+export const calculateMinimumReturn = ({
+    toToken,
+    toAmount,
+    slippage,
+}: {
+    toToken: FungibleTokenDetailed
+    toAmount: string | undefined
+    slippage: number
+}): string => {
+    const toWei = new BigNumber(toAmount || '0').multipliedBy(pow10(toToken.decimals ?? 0))
+    const slippageWei = new BigNumber(slippage).dividedBy(BIPS_BASE)
+    const minReturnWei = toWei.times(new BigNumber(1).minus(slippageWei))
```

Comment:
```suggestion
    const minReturnWei = toWei.times(ONE.minus(slippageWei))
```

https://github.com/DimensionDev/Maskbook/blob/ae381791ef2a8347cec10d659440721cc04c8f24/packages/web3-shared/src/utils/number.ts#L4

Reply:
fixed